infra/modules/gcp-psoxy-rest/main.tf

```diff
@@ -106,20 +106,18 @@ resource "google_cloudfunctions2_function" "function" {
   ]
 }
 
-resource "google_cloudfunctions2_function_iam_member" "invokers" {
-  for_each = toset(var.invoker_sa_emails)
-
-  cloud_function = google_cloudfunctions2_function.function.id
-  member         = "serviceAccount:${each.value}"
-  role           = "roles/cloudfunctions.invoker"
-}
-
-resource "google_cloudfunctions2_function_iam_member" "testers" {
-  for_each = toset(var.gcp_principals_authorized_to_test)
-
-  cloud_function = google_cloudfunctions2_function.function.id
-  member         = each.value
-  role           = "roles/cloudfunctions.invoker"
+# bizarrely, `google_cloudfunctions2_function_iam_binding` doesn't work for this; wtf?
```

Comment:
isn't working even with `roles/run.invoker` role, right ?

Reply:
the `google_cloudfunctions2_function_iam_binding` doesn't work. I had to use the other resource.